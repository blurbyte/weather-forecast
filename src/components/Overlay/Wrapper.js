import styled from 'styled-components';

import { simpleFadeIn } from '../../styles/animations';

const Wrapper = styled.div`
  position: fixed;
  z-index: 9999;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  background: rgba(0, 0, 0, 0.75);
  animation: ${simpleFadeIn} 0.15s ease-out forwards;
  opacity: 0;
`;

export default Wrapper;
