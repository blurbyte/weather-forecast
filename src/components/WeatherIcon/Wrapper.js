import styled from 'styled-components';

import media from '../../styles/media';

const Wrapper = styled.div`
  width: ${props => (props.small ? '5rem' : '11rem')};
  height: ${props => (props.small ? '5rem' : '11rem')};

  ${media.phone`
    width: ${props => props.small && '3.6rem'};
    height: ${props => props.small && '3.6rem'};
  `};
`;

export default Wrapper;
