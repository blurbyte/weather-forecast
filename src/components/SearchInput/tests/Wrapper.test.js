import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Wrapper from '../Wrapper';

test('renders correctly', () => {
  const component = renderer.create(<Wrapper />);
  expect(component.toJSON()).toMatchSnapshot();
});
