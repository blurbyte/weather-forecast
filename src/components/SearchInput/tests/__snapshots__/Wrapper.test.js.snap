// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  position: relative;
}

<div
  className="c0"
/>
`;
